File: Front-end/src/features.js
import flagsmith from 'flagsmith';

await flagsmith.init({
    environmentID: 'jtDu5Dr22myxztrApPfUNH',
    identity: Math.random(),
    cacheFlags: false,
    enableAnalytics: true,
});

export function isImportTransactionsEnabled () {
    return flagsmith.hasFeature('import-transactions');
}

export function getDashboardValue () {
    return flagsmith.getValue('dashboard');
}

File: Back-end/features.py
from flagsmith import Flagsmith

flagsmith = Flagsmith(
    environment_key="jtDu5Dr22myxztrApPfUNH",
)

def is_import_transactions_enabled():
    flags = flagsmith.get_environment_flags()

    return flags.is_feature_enabled("import-transactions")